db/schema.rb

```diff
@@ -59,6 +59,31 @@
     t.index ["ordering"], name: "index_attachments_on_ordering"
   end
 
+  create_table "call_for_evidence_participations", id: :integer, charset: "utf8mb3", force: :cascade do |t|
+    t.integer "edition_id"
+    t.string "link_url"
+    t.datetime "created_at", precision: nil
+    t.datetime "updated_at", precision: nil
+    t.string "email"
+    t.integer "call_for_evidence_response_form_id"
+    t.text "postal_address"
+    t.index ["call_for_evidence_response_form_id"], name: "index_cons_participations_on_cons_response_form_id"
```

Comment:
Same problem as above - "cons" in index name

Reply:
fixed